### PR TITLE
Add missing js data elements

### PR DESCRIPTION
**Pull request by SID177**
Closes https://github.com/rtCamp/rtMedia/issues/311

### CODE REVIEW

File: templates/main.php [hunk 1]
```diff
@@ -96,7 +96,7 @@
 	 */
 	do_action( 'bp_before_group_home_content' ); ?>
 
-	<div id="item-header" role="complementary">
+	<div id="item-header" role="complementary" data-bp-item-id="<?php bp_group_id(); ?>" data-bp-item-component="groups" class="groups-header single-headers">
```
Comment (abhijitrakas) [inline templates/main.php]: Could we use `esc_attr` for `bp_group_id()`?

Comment (SID177) [reply templates/main.php]: Its internally escaped, that's why I haven't echoed it here

Comment (abhijitrakas) [reply templates/main.php]: Ok

**Comment by abhijitrakas** [comment]
LGTM! 👍

**Pull request merged**